README.md

```diff
@@ -185,7 +366,7 @@ We could alternatively have established the relationship by adding a student to
 subject's list of students:
 
 ```java
-// create students >-< subjects association
+// create students <> subjects association
```

Comment:
I can't comment on the exact line so it's not actually part of the diff, but a few lines down, it says
```We only need to add to one list to add the association in the database.```

But it looks like the code adds to both lists (addStudent method calls the addSubject method).

Reply:
Removed the comment about adding to one list.